### ISSUE TITLE
Should Authentication-Results be standards track?

### ISSUE THREAD

**Issue by ietf-svn-bot** [ietf-wg-dmarc/dmarc-draftissues#89]
`resolution_out-of-scope` `type_defect`   |    by mike@mtcc.com

___

It was pointed out that it's not clear whether Authentication-Results should be standards track or not. That is, is there a consumer for the results that needs protocol agreement. The DKIM and SPF results generally contain enough information to that a consumer of auth-res so as not to need to re-run DKIM or SPF checks if they are in the same administrative domain. The same is not true of DMARC. See issue on scaling for why that is bad. Since Authentication-Results is officially without a working group, it should be decided what its status is: is it a protocol that should be standards track? is it just informational debugging information for humans? Should it just be declared historic since it isn't officially maintained?

___
_Issue migrated from trac:89 at 2022-01-24 16:52:30 +0000_

**Comment by ietf-svn-bot** [comment]
_@mike@mtcc.com_ _changed priority from `minor` to `major`_

**Comment by ietf-svn-bot** [comment]
_@dougfoster.emailstandards@gmail.com_ _commented_


___
RE this ticket and related ticket #90:
In ticket #95, I argue that Authentication-Results should be fully deprecated in favor of ARC sets, contingent on changing the ARC specification to allow multiple ARC sets within a single ADMD.

**Comment by ietf-svn-bot** [comment]
_@johnl@taugh.com_ _changed status from `new` to `closed`_

**Comment by ietf-svn-bot** [comment]
_@johnl@taugh.com_ _set resolution to `out-of-scope`_

**Comment by ietf-svn-bot** [comment]
_@johnl@taugh.com_ _commented_


___
we are not revising the A-R RFCs
